app/src/main/java/com/github/se/cyrcle/MainActivity.kt

```diff
@@ -37,12 +38,14 @@ class MainActivity : ComponentActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+    requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
Is that something you added, if so why ?

Reply:
As mentioned in the PR description, this allows us to fix the orientation of the app so users don't accidentally go into landscape mode, since our app doesn't support landscape mode nor should it

Reply:
Ok i wasnt aware that the team decided that the app wouldnt support landscape. But it makes sense since bikers typically dont ride with their ipad in their hand ! 
It's just that some screens could technically support landscape mode / it would be cool (list screen is trivial and could be useful if you want to see a bit bigger text for blind people like me), so I am not a big fan of a generalized lock.

Dont worry I let that slide im just being annoying

Reply:
I'll make sure it's again mentioned at tomorrow's meeting so we can make a final decision